src/components/sendV2/send.js

```diff
@@ -1,35 +1,45 @@
 import React from 'react';
 import MultiStep from './../multiStep';
 import Form from './form';
+import Summary from './summary';
 import { parseSearchParams } from './../../utils/searchParams';
 import styles from './send.css';
 
 class Send extends React.Component {
   constructor(props) {
     super(props);
 
-    this.getSearchParams = this.getSearchParams.bind(this);
+    this.state = {
+      fields: {},
+    };
   }
 
-  getSearchParams() {
-    return parseSearchParams(this.props.history.location.search);
+  componentDidMount() {
+    const { recipient, amount, reference } = parseSearchParams(this.props.history.location.search);
+
+    // istanbul ignore else
+    if (recipient && amount) {
```

Comment:
Shouldn't be a logical AND, it should be able to fill only the recipient OR amount with data from URL